components/application-configuration/api/src/main/resources/ApplicationResources.properties

```diff
@@ -20,3 +20,4 @@ PhenoTips.LabeledIdentifierSettings_allowDuplicateEids=Allow duplicate external
 PhenoTips.LabeledIdentifierGlobalSettings_allowOtherEids=Allow other external identifiers
 PhenoTips.LabeledIdentifierGlobalSettings_hideInternalIds=Show external identifiers (when available) instead of internal ones in the Patient and Family records
 PhenoTips.LabeledIdentifierGlobalSettings_externalIdFormat=External id display format
+PhenoTips.LabeledIdentifierGlobalSettings_validateExternalId=Check uniqueness of external ID
```

Comment:
Maybe "Require unique external identifiers" is a bit clearer?

Reply:
comment addressed